packages/nimble-components/build/generate-workers/source/types.ts

```diff
@@ -20,3 +20,38 @@ export interface Dimensions {
     width: number;
     height: number;
 }
+
```

Comment:
I'll be OOO for the next week starting tomorrow but discussed some high level feedback with @atmgrifter00 that would be worth having a discussion on:
- Can we avoid all the optional types below in RenderConfig and TransformConfig
- Can we keep a bunch of the new state added to the top-level wafer map in smaller classes that encapsulate their algorithms that are later accessed by createSnapshot instead of putting them all on top-level wafermap
- Corresponding to the above, there are a lot of non-null assertions being added, can we avoid adding more and start cleaning some up?
- Is there a plan for addressing the intermittent test bug: https://github.com/ni/nimble/issues/2104
  - It seems likely that the test became intermittent due to this recent change: https://github.com/ni/nimble/pull/2095

Reply:
I changed the intermittent test to check for NaN instead of a specific value